e2e/cypress/integration/02_global_filter.spec.js

```diff
@@ -0,0 +1,76 @@
+describe('global projects filter', () => {
+  const nonAdminUsername = "cypressTestUserNonAdmin"
+  const proj1  = "cypress-project-1"
+  const proj2  = "cypress-project-2"
+  const pol_id = "cypress-policy"
+
+  before(() => {
+    cy.adminLogin('/').then(() => {
+      let admin = JSON.parse(localStorage.getItem('chef-automate-user'))
+
+      cy.createProject(admin.id_token, proj1)
+      cy.createProject(admin.id_token, proj2)
+      cy.createUser(admin.id_token, nonAdminUsername)
+      cy.createPolicy(admin.id_token, "cypress-policy-id", nonAdminUsername, [proj1, proj2])
+      cy.logout()
+    })
+  })
+
+  after(() => {
+    cy.adminLogin('/').then(() => {
+      let admin = JSON.parse(localStorage.getItem('chef-automate-user'))
+      cy.cleanupProjects(admin.id_token)
+      cy.cleanupUsers(admin.id_token)
+      cy.request({
+        auth: { bearer: admin.id_token },
+        method: 'DELETE',
+        url: `/apis/iam/v2beta/policies/${pol_id}`,
+        failOnStatusCode: false
+      }).then((response) => {
+        expect([200, 404]).to.include(response.status)
+      })
+    })
+  })
+
+  it('shows all projects for admin', () => {
+    cy.adminLogin('/')
+    cy.visit('/settings')
+    // hide modal unrelated to test flow
+    cy.get('app-welcome-modal').invoke('hide')
+
+    cy.get('chef-sidebar')
+      .should('have.attr', 'minor-version')
+      .then((version) => {
+        if (version === 'v1') {
```

Comment:
Should we/could use this conditional at the top of the describe to not run the tests (or setup) at all if we're not in 2.1? 🤔

Reply:
Definitely. 👍